waku/v2/protocol/rln/waku_rln_relay.go

```diff
@@ -270,7 +271,7 @@ func (rlnRelay *WakuRLNRelay) Validator(
 
 			return false
 		default:
-			log.Debug("unhandled validation result", zap.Int("validationResult", int(validationRes)))
+			log.Info("unhandled validation result", zap.Int("validationResult", int(validationRes)))
```

Comment:
This one i think makes sense for it to be Error!

Reply:
Fixed at https://github.com/waku-org/go-waku/pull/1003/commits/5c90a83b2b85198d21d7ef1d9684f65df59dac41